README.md

```diff
@@ -127,7 +127,7 @@ compile_gresources(<output>
 - **SOURCE_DIR** ***resource_directory***
```

Comment:
commit message:
could you remove the trailing period? :)
( I know this is a bit nit-picky, but I'm currently working on https://github.com/coala/coala which is all about code quality, involving good commit messages :3)